src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/MetricsEventSource.cs

```diff
@@ -105,9 +105,9 @@ public void CollectionStop(string sessionId, DateTime intervalStartTime, DateTim
         [Event(4, Keywords = Keywords.TimeSeriesValues)]
```

Comment:
One last bit I missed, we should add `Version=1` (0 is default) to the EventAttribute on the events that gained an extra parameter. It helps parsers know whether the extra data will be there.